src/components/accordion/useAccordion.tsx

```diff
@@ -1,17 +1,17 @@
 import * as React from 'react';
-import { PUITheme, ThemeMode } from 'utils/types';
 import { useCombinedRefs } from 'utils/hooks/combinedrefs';
+import { PUITheme, ThemeMode } from '../../utils/types';
```

Comment:
You, can create index file inside utils where you export everything and then you can write import { useCombinedRefs, PUITheme, ThemeMode } from 'utils'; And why for one case you use relative path, for another absolute?

Reply:
for correct using types from UI in typescript project. 
after build and download package from npms in node_modules inside the package you should to use absolute paths - idk why but for types for sure

Reply:
it's good approach but it doesn't work for ts lib as far as I know